### DISCUSSION TITLE
How do 'estimatedWidth' and 'priority' attributes work?

### DISCUSSION THREAD

**Discussion by WFZ1** [epam/UUI#434] [Q&A] [answered]
I often see different priority numbers and 100500 in the example. Could you tell how it affects the components with this attribute? Also the similar question about 'estimatedWidth'.
Thanks for help in advance)

**Comment by jakobz** [comment, marked as answer]
Hi! Both properties affects how menu items collapse into "more" and burger, in case they doesn't fit.
EstimatedWidth should specify width of the menu item. We ask to set this manually, as measuring items programmatically is tricky, and cause double-rendering.
Priority defines the order in which items collapse. Items with lower priority would collapse first. You can set equal priorities to collapse certain items simultaneously.
100500 is a number we often use to set some really big number. It's originates from a meme.

**Reply by WFZ1** [reply]
Thank you for detail answer
